testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/UserTest.java

```diff
@@ -3003,6 +3003,20 @@ private GroupRepresentation createGroup(RealmResource realm, GroupRepresentation
         return group;
     }
 
+    private GroupRepresentation createSubGroup(RealmResource realm, String parentGroupId, GroupRepresentation group) {
+        final String groupId;
+        try (Response response = realm.groups().group(parentGroupId).subGroup(group)) {
+            groupId = ApiUtil.getCreatedId(response);
+            getCleanup().addGroupId(groupId);
+        }
+
+        assertAdminEvents.assertEvent(realmId, OperationType.CREATE, AdminEventPaths.groupSubgroupsPath(parentGroupId), group, ResourceType.GROUP);
+
+        // Set ID to the original rep
+        group.setId(groupId);
+        return group;
+    }
+
     @Test
```

Comment:
This is not used anymore, could you please remove this method?

Reply:
Done.